youtube_dl/extractor/adn.py

```diff
@@ -189,19 +203,15 @@ def _real_extract(self, url):
                     for f in m3u8_formats:
                         f['language'] = 'fr'
                 formats.extend(m3u8_formats)
-        if not error:
-            error = options.get('error')
-        if not formats and error:
-            raise ExtractorError('%s said: %s' % (self.IE_NAME, error), expected=True)
         self._sort_formats(formats)
 
         return {
             'id': video_id,
             'title': title,
-            'description': strip_or_none(metas.get('summary') or video_info.get('resume')),
+            'description': strip_or_none(metas.get('summary')),
             'thumbnail': video_info.get('image'),
             'formats': formats,
-            'subtitles': self.extract_subtitles(sub_path, video_id),
+            'subtitles': sub_path and self.extract_subtitles(sub_path, video_id),
             'episode': metas.get('subtitle') or video_info.get('videoTitle'),
             'series': video_info.get('playlistTitle'),
         }
```

Comment:
additional info can be extracted from `video` request.

Reply:
Just `duration`, right?

Reply:
non, there is `season_number`, `episode_number`, `timestamp`, etc...

Reply:
Huh, I don't see it. Can you give me an example for a video with additional information in either of both JSON responses?

Reply:
example for the first test case:
https://gw.api.animedigitalnetwork.fr/video/7778

Reply:
Ah, thanks!